count_cells.py

```diff
@@ -112,15 +111,24 @@ def get_project_cell_count(self, accession_id: str) -> Union[int, None]:
         if not os.path.isdir(f'projects/{accession_id}'):
             logging.warning('Unable to find path %s', f'projects/{accession_id}')
             return None
-        total_count = 0
+        cell_counts = {}
         for file in Path(f'projects/{accession_id}/matrices').glob('**/*'):
-            if file.is_file() and file.name.endswith(('.mtx', '.mtx.gz')):
-                cell_count = self.count_cells(file)
-                logging.info('Cell count in %s is %s', file, cell_count)
-                if cell_count is not None:
-                    total_count += cell_count
-        logging.info('Total cell count in %s is %s', accession_id, total_count)
-        return total_count
+            if file.is_file():
+                if file.parent.name not in cell_counts:
+                    cell_counts[file.parent.name] = {'mtx': 0, 'barcodes': 0}
+                if file.name.endswith('.mtx.gz'):
+                    cell_count = self.count_cells(file)
+                    logging.info('Cell count in %s is %s', file, cell_count)
+                    if cell_count is not None:
+                        cell_counts[file.parent.name]['mtx'] += cell_count
+                elif file.name in ('barcodes.gz', 'barcodes.tsv.gz'):
```

Comment:
A missing `.tsv` in the file barcodes file name is actually an error. 

https://github.com/DailyDreaming/load-project/issues/132

Not sure why the choice was made to code around the error rather than to fix it.